xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -71,7 +73,13 @@
      * @see #getSection()
      */
     private String section;
-
+    
+    /**
+     * @see #isExcludeFirstHeading()
+     */
+    private boolean getExcludeFirstHeading;
```

Comment:
this is not what @tmortagne said...

Reply:
Why are you naming a field with a "get" prefix. That's definitely not a good practice!

Reply:
Sorry, my mistake :( Simple `excludeFirstHeading` will be better as similar to other fields names. Changing it.

Reply:
Now it makes sense. Thankyou :)